scripts/generate-tutorial-variant-map.ts

```diff
@@ -0,0 +1,105 @@
+import createFetch from '@vercel/fetch'
+import path from 'path'
+import fs from 'fs'
+const fetch = createFetch()
+
+const MAX_LIMIT = '100' // Defaults to API max
+
+async function fetchAll({ after, fetchedTutorials }) {
+	// Set the base array if it's the first call
+	if (typeof fetchedTutorials === 'undefined') {
+		fetchedTutorials = []
+	}
```

Comment:
Will be refactoring this to use the new variants query param on `/tutorials` once released. We won't do batch requests here via pagination since it will be well below the 100 limit